identity/aziot-identityd/src/lib.rs

```diff
@@ -537,9 +537,17 @@ impl Api {
         self.local_identities = local_modules;
         self.settings = settings;
 
-        let _ = self
+        // Attempt to re-provision the device. Failures need to be logged and the device should
+        // run offline.
+        if let Err(err) = self
             .reprovision_device(auth::AuthId::LocalRoot, trigger)
-            .await?;
+            .await
+        {
+            log::warn!(
+                "Failed to reprovisioning device. Running offline. Reprovisioning failure reason: {}. ",
```

Comment:
`Failed to reprovisioning` -> `Failed to reprovision`

Reply:
Fixed